spacemesh/v2alpha1/v2alpha1.proto

```diff
@@ -13,4 +13,9 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
       url: "https://spacemesh.io/";
     };
   };
-};
\ No newline at end of file
+};
+
+enum SortOrder {
+  ASC = 0;
+  DESC = 1;
+}
```

Comment:
Two questions:
- what about "don't sort" variant? It could save some cycles when sorting is not needed.
- does the default need to be "sorted"?

Reply:
We can add third option which is "don't sort" but the current implementation sorts in ascending order, and I don't want to break existing integrations for others.